src/custom/components/Header/URLWarning/index.tsx

```diff
@@ -0,0 +1,9 @@
+import { PRODUCTION_URL } from '@src/custom/constants'
```

Comment:
`@src`? why?

Reply:
good catch, it was vscode auto-importing